### DISCUSSION TITLE
Opening JSON files with Version 9.1 Update 1

### DISCUSSION THREAD

**Discussion by choitingy** [Open-Systems-Pharmacology/Forum#987] [Questions & Problems] [answered]
I am trying to open JSON files into PK-Sim Version 9.1 - Update 1. I select File > Load from Snapshot, and a window opens for me to Start Import with a Log Level of "Information". After the progress bar finishes loading on the lower right hand corner, the window remains grey and after exiting it there is nothing in the Building Blocks or Simulations. Might I be missing something here?

**Comment by msevestre** [comment, marked as answer]
@choitingy Can you update a screenshot? If the snapshot file is `big`, PK-Sim might not be done with computing and you are closing the windows too early.

**Reply by choitingy** [reply]
@msevestre You were right, the file was just very big. I was confused (and impatient!) with the pauses between the Creating Individuals, Creating Compounds, etc. progress bars. Thanks!
